website/content/docs/v0.3/Tutorial/prereq-kubernetes.md

```diff
@@ -0,0 +1,76 @@
+ - a cluster running inside a virtual machine stack such as VMWare
+ - a Talos Kubernetes cluster running on a spare machine
+
+Two important things are needed in this cluster:
+- Kubernetes `v1.18` or later
+- Ability to expose tcp and udp Services to the workload cluster machines
+
+For the purposes of this tutorial, we will create this cluster in Docker on a
+workstation, perhaps a laptop.
+
+If you already have a suitable Kubernetes cluster, feel free to skip this step.
+
+## Create a Local Management Cluster
+
+The `talosctl` CLI tool has built-in support for spinning up Talos in docker containers.
+Let's use this to our advantage as an easy Kubernetes cluster to start from.
+
+Issue the following to create a single-node Docker-based Kubernetes cluster:
+
+```bash
+export HOST_IP="192.168.1.150"
+
+talosctl cluster create \
+  --name sidero-demo \
+  -p 69:69/udp,8081:8081/tcp \
+  --workers 0 \
+  --endpoint $HOST_IP
+```
+
+The `192.168.1.150` IP address should be changed to the IP address of your Docker
+host.  This is _not_ the Docker bridge IP but the standard IP address of the
+workstation.
+
+Note that there are two ports mentioned in the command above.  The first (69) is
+for TFTP.  The second (8081) is for the web server (which serves netboot
+artifacts and configuration).
+
+Exposing them here allows us to access the services that will get deployed on this node.  In turn, we will be running our Sidero services with `hostNetwork: true`, so the Docker host will forward these to the Docker container, which will in turn be running in the same namespace as the Sidero Kubernetes components.  A full separate management cluster will likely approach this differently, with a load balancer or a means of sharing an IP address across multiple nodes (such as with MetalLB).
+
+## Access the cluster
+
+Once the cluster create command is complete, you can retrieve the kubeconfig for it using the Talos API:
+
+```bash
+talosctl kubeconfig
+```
+
+> Note: by default, Talos will merge the kubeconfig for this cluster into your
+> standard kubeconfig under the context name matching the cluster name your
+> created above.  If this name conflicts, it will be given a `-1`, a `-2` or so
+> on, so it is generally safe to run.
+> However, if you would prefer to not modify your standard kubeconfig, you can
+> supply a directory name as the third parameter, which will cause a new
+> kubeconfig to be created there instead.
+> Remember that if you choose to not use the standard location, your should set
+> your `KUBECONFIG` environment variable or pass the `--kubeconfig` option to
+> tell the `kubectl` client the name of the `kubeconfig` file.
+
+## Untaint Control Plane
+
+Because this is a single node cluster, we need to remove the "NoSchedule" taint on the node to make sure non-controlplane components can be scheduled.
+
+```bash
+kubectl taint node talos-default-master-1 node-role.kubernetes.io/master:NoSchedule-
```

Comment:
we can remove this step if we add the following argument to the `talosctl cluster create` above:

```
 --config-patch '[{"op": "add", "path": "/cluster/allowSchedulingOnMasters", "value": true}]'
 ```